charts/external-dns/crds/dnsrecord.yaml

```
@@ -0,0 +1,97 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

```suggestion
apiVersion: apiextensions.k8s.io/v1
```
Nit, we only have document separators where required.